src/main/java/edu/hm/hafner/analysis/registry/BluePearlDescriptor.java

```diff
@@ -0,0 +1,23 @@
+package edu.hm.hafner.analysis.registry;
+
+import edu.hm.hafner.analysis.IssueParser;
+import edu.hm.hafner.analysis.parser.BluePearlParser;
+
+/**
+ * A descriptor for the Blue Pearl Visual Verification Suite.
+ *
+ * @author Simon Matthews
+ */
+class BluePearlDescriptor extends ParserDescriptor {
+    private static final String ID = "bluepearl";
```

Comment:
Since you capitalized the parser `BluePearl` it might make sense to use as ID `blue-pearl`. But I leave that up to you...